cds_ils/cli.py

```diff
@@ -130,6 +195,16 @@ def run_command(command, catch_exceptions=False, verbose=True):
     run_command("access allow superuser-access role admin")
     run_command("access allow ils-backoffice-access role librarian")
 
+    patrons = User.query.all()
+    indexer = current_app_ils.patron_indexer
+
+    click.secho("Now indexing {0} patrons".format(len(patrons)), fg="green")
+
+    Patron = current_app_ils.patron_cls
+    for pat in patrons:
+        patron = Patron(pat.id)
+        indexer.index(patron)
+
 
```

Comment:
you can use instead
`run command("patrons index")` 
to be checked if it works!